### PR TITLE
Add base tokens from coinpaprika

### PR DESCRIPTION
**Pull request by tiagofilipenunes**
Add BASE tokens to price_base_tokens.sql from [Coinpaprika](https://coinpaprika.com/tag/base-token/home-overview/?all=true).

### CODE REVIEW

**Comment by jeff-dude** [comment]
please see error in CI test and push commits to fix:
https://github.com/duneanalytics/spellbook/actions/runs/7918639185/job/21617537009?pr=5353#step:11:30

File: models/prices/base/prices_base_tokens.sql [hunk 1]
```diff
@@ -29,7 +29,40 @@ FROM
...
+    ('sdex-smardex', 'base', 'SDEX', 0xfd4330b0312fdeec6d4225075b82e00493ff2e3f, 18),
+    ('tbtc-tbtc', 'base', 'TBTC', 0x236aa50979d5f3de3bd1eeb40e81137f22ab794b, 18),
+    ('mbs-monkeyleague', 'base', 'MBS', 0x8Fbd0648971d56f1f2c35Fa075Ff5Bc75fb0e39D, 18),
+    ('isk-iskra-token', 'base', 'ISK', 0x17d2628d30f8e9e966c9ba831c9b9b01ea8ea75c, 18),
+    ('vela-vela-token', 'base', 'VELA', 0x5a76a56ad937335168b30df3aa1327277421c6ae, 18),
+    ('tower-tower', 'base', 'TOWER', 0xf7C1CEfCf7E1dd8161e00099facD3E1Db9e528ee, 18),
+    ('frame-frametoken', 'base', 'FRAME', 0x91F45aa2BdE7393e0AF1CC674FFE75d746b93567, 18),
+    ('omni-omnicat', 'base', 'OMNI', 0xc48e605c7b722a57277e087a6170b9e227e5ac0a, 18),
+    ('bsx-basex', 'base', 'BSX', 0xd5046b976188eb40f6de40fb527f89c05b323385, 18),
+    ('fxdx-fxdx', 'base', 'FXDX', 0x30b593f8c3ab37615359b4e0e6df2e06d55bb55d, 18),
```
Comment (jeff-dude) [inline models/prices/base/prices_base_tokens.sql:46]: note this is returning a failure for invalid id:
https://github.com/duneanalytics/spellbook/actions/runs/7918639192/job/21617536821?pr=5353#step:6:47

Comment (NIXBNT) [reply models/prices/base/prices_base_tokens.sql:46]: Looks like it should be `fxdx-fxdx-exchange` as the ticker

**Review by jeff-dude — APPROVED** [review]
thanks for the quick fixes!

**Pull request merged**